doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -427,15 +427,18 @@ \subsubsection{Built-in types}
   \Strut
   Type & Binary  & Decimal \\
   \hline
-  $\ty{integer}$     & $\bits{0000}$ & 0 \\
-  $\ty{bytestring}$  & $\bits{0001}$ & 1 \\
-  $\ty{string}$      & $\bits{0010}$ & 2 \\
-  $\ty{unit}$        & $\bits{0011}$ & 3 \\
-  $\ty{bool}$        & $\bits{0100}$ & 4 \\
-  $\ty{list}$        & $\bits{0101}$ & 5 \\
-  $\ty{pair}$        & $\bits{0110}$ & 6 \\
-  (type application) & $\bits{0111}$ & 7 \\
-  $\ty{data}$        & $\bits{1000}$ & 8 \\
+  $\ty{integer}$                 & $\bits{0000}$ & 0  \\
+  $\ty{bytestring}$              & $\bits{0001}$ & 1  \\
+  $\ty{string}$                  & $\bits{0010}$ & 2  \\
+  $\ty{unit}$                    & $\bits{0011}$ & 3  \\
+  $\ty{bool}$                    & $\bits{0100}$ & 4  \\
+  $\ty{list}$                    & $\bits{0101}$ & 5  \\
+  $\ty{pair}$                    & $\bits{0110}$ & 6  \\
+  (type application)             & $\bits{0111}$ & 7  \\
+  $\ty{data}$                    & $\bits{1000}$ & 8  \\
+  $\ty{bls12\_381\_G1\_element}$ & $\bits{1001}$ & 9  \\
```

Comment:
It's a bit annoying that we have to update this appendix separately, and it's maybe not clear how serialisation should work in the different versions of Plutus which have different builtins available. . We could maybe try make this generic over the types/functions and shift the specifications for the individual types into the appendices describing the individual builtins, but maybe that would fragment it too much. The encodings and decodings for things like integers and bytestrings are fundamental, and they really do have to be in this section.